docs/en/stack/ml/anomaly-detection/geographic-anomalies.asciidoc

```diff
@@ -218,6 +219,12 @@ image::images/weblogs-anomaly-explorer-geopoint.jpg[A screenshot of an anomalous
 You can use the top influencer values to further filter your results and
 identify possible contributing factors or patterns of behavior.
 
+You can also view the anomaly in **Maps** by clicking on the "View in Maps" 
+option under the **Actions** menu in the anomaly table.
```

Comment:
Our "Writing about the user interface" guidelines refer to this simply as the "action menu", so that's likely simplest here too:
```suggestion
in the action menu in the anomaly table.
```